dashboards/CockroachDB/CockroachDB-Details-Storage.json

```diff
@@ -117,11 +151,11 @@
           "step": 2
         },
         {
-          "expr": "sum(sum(capacity{job=\"cockroachdb\",cluster=\"$cluster\",instance=~\"$node\"}) by (instance))",
+          "expr": "sum(sum(capacity{dcos_component_name=\"CockroachDB\",host=~\"$node\"}) by (host))",
           "format": "time_series",
           "interval": "",
           "intervalFactor": 2,
-          "legendFormat": "Capacity",
+          "legendFormat": "Storage Capacity",
```

Comment:
This plot is titled Storage Capacity, so this label could be `Total`, compared to the other label `Used`.